src/extension.ts

```diff
@@ -74,7 +74,7 @@ export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(PartEditorProvider.register(context));
   context.subscriptions.push(PartGraphSelPanel.register(context));
 
-  context.subscriptions.push(CircleViewerProvider.register(context));
+  CircleViewerProvider.register(context);
```

Comment:
A little bit of improvement is that it does registration at one place. (not passing the provider disposable to the caller)